tekton/pipeline.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
```

Comment:
So the general idea is that different triggers in the releases repo will to through different tekton pipelines, right? Which will probably share most of the tasks because the only difference between them would be how to run the tests?

Reply:
Yes - only the tasks running the different tests would be different. Other than that the same tasks for cluster creation / tear down are used.